### ISSUE TITLE
Fixate behavior if not all fonts from manifest was found

### ISSUE THREAD

**Issue by CAMOBAP** [fontist/fontist#189]
### Intro

Per @alexeymorozov for now if some font from manifest will not be found, `fontist` will return an empty path to a font file.

Example:

```

Fontist::Manifest::Locations.from_hash({
  "Known Font" => nil,
  "Unknown Font" => nil,
})

# will produce without throwing Fontist::Errors::NonSupportedFontError exception 

{
  "Known Font"=> {
    "Regular" => { 
      "full_name"=>"Known Font",
       "paths"=>["/Users/user/.fontist/fonts/KnownFont.TTF"]
    },
    "Bold"=>{
      "full_name"=>"Known Font Bold",
      "paths"=>["/Users/user/.fontist/fonts/KnownFontB.TTF"]
    }
  },
  "Unknown Font"=> {
    "Regular"=>{
      "full_name"=>nil,
      "paths"=>[]
    }
  }
}
```

### Expected behavior

I'm not 100% but I think we in our logic we expect to get font installation error as son as possible to make it clear for users https://github.com/metanorma/metanorma-standoc/pull/392/files#diff-1f5de39b7fe64246ff7b8f10d799e6e1474f799ef5895da3202937d85963300eL128

@ronaldtse please share your thoughts

**Comment by ronaldtse** [comment]
Agree that some error should be raised. Thoughts @alexeymorozov ? Do we want both behaviors for the Fontist user, such that the call can fail explicitly or silently (e.g. `silent: true`)?

**Comment by alexeymorozov** [comment]
Seems it's more expected to get an exception in such cases, so it's ok with me.

There is also another exception that could happen when no path is returned: if license is not approved with neither a flag, nor UI. I think, it would be consistent to raise it too. It's name `Fontist::Errors::LicensingError`.

**Comment by ronaldtse** [comment]
@alexeymorozov agree. Let's raise errors on both cases. Please help make the issues and corresponding PRs. Thanks!

**Comment by alexeymorozov** [comment]
Ok :) Created an issue #193

**Comment by ronaldtse** [comment]
Thanks @alexeymorozov !

**Comment by CAMOBAP** [comment]
Closing this one because a separate issue created

**Comment by alexeymorozov** [comment]
Released in v1.8.1
